style/typography.js

```diff
@@ -35,24 +31,46 @@ const H1 = styled.h1`
 `;
 
 const H2 = styled.h2`
-  font-size: ${fontSizes.h2};
   font-family: ${fonts.swearDisplay};
   font-weight: ${fontWeights.medium};
   color: ${colors.BURNTORANGE};
+  font-size: 2em;
+  @media ${min.tablet} {
+    font-size: 2.6em;
+  }
 `;
 
 const H3 = styled.h3`
-  font-size: ${fontSizes.h3};
+  font-size: clamp(1.5em, 6vw, 2em);
   font-family: ${fonts.swearDisplay};
   font-style: ${fontStyles.italic};
   font-weight: ${fontWeights.medium};
 `;
 
+const H4 = styled.h3`
+  font-family: ${fonts.poppins};
+  font-weight: ${fontWeights.medium};
+  color: ${colors.BROWN};
+  letter-spacing: 0.1em;
+  font-size: 1em;
+`;
+
 const P = styled.p`
   font-family: ${fonts.poppins};
   font-weight: ${fontWeights.normal};
   font-size: 1em;
   line-height: 1.8em;
 `;
 
-export { H1, H2, H3, P, fontWeights, fonts };
+const LinkText = styled(P)`
```

Comment:
It's not exactly a Link, perhaps a "InputInfoText" or something would be better?